TradeItIosTicketSDK2/TradeItNumericToggleInputCell.swift

```diff
@@ -0,0 +1,83 @@
+import UIKit
+
+class TradeItNumericToggleInputCell: UITableViewCell {
+    @IBOutlet weak var textField: TradeItNumberField!
+    @IBOutlet weak var quantityTypeButton: UIButton!
+
+    var onValueUpdated: ((_ newValue: NSDecimalNumber?) -> Void)?
+    var onQuantityTypeToggled: (() -> Void)?
+
+    override func awakeFromNib() {
+        self.textField.padding = UIEdgeInsets(top: 0, left: 5, bottom: 0, right: 5)
```

Comment:
Can this be done in the interface builder? Seems more consistent with our other screens/nibs if so.

Reply:
I just removed this. Seems like it was not needed.